docker/compose/docker-compose.yml

```diff
@@ -15,23 +18,54 @@ services:
       POSTGRES_USER:
       POSTGRES_PASSWORD:
       POSTGRES_DB:
+      GITPLAG_URL: "http://gitplag:8090"
```

Comment:
Maybe it should be `${DEPLOYMENT_URL}:8090` as the bottom line?

Reply:
At first I though the same way as you but it turned out this url is correct.

The difference is that `GITPLAG_URL` is used from the `flaxo-backend` container where `DEPLOYMENT_URL` may not make sense. F.e. if `DEPLOYMENT_URL` is `http://localhost` when `flaxo-backend` will try to request its own container's localhost. 

Differently `GITPLAG_UI_URL` is used to access gitplag from outer network by application users where `DEPLOYMENT_URL` does make sense.

Reply:
ok